### ISSUE TITLE
GetLLMProvider exception issue

### ISSUE THREAD

**Issue by sobit-nep** [krishnaik06/CrewAI-Projects#1]
I'm getting this issue. How can i solve it?
File "c:\Users\50bit\.conda\envs\venv\lib\site-packages\litellm\litellm_core_utils\get_llm_provider_logic.py", line 515, in get_llm_provider    raise litellm.exceptions.BadRequestError(  # type: ignore
litellm.exceptions.BadRequestError: litellm.BadRequestError: GetLLMProvider Exception - 'ChatGoogleGenerativeAI' object has no attribute 'split'

original model: verbose=True model='models/gemini-1.5-flash' google_api_key=SecretStr('**********') temperature=0.5 client=<google.ai.generativelanguage_v1beta.services.generative_service.client.GenerativeServiceClient object at 0x000002289050DAB0> default_metadata=()

**Comment by Obsidianage** [comment]
> I'm getting this issue. How can i solve it? File "c:\Users\50bit.conda\envs\venv\lib\site-packages\litellm\litellm_core_utils\get_llm_provider_logic.py", line 515, in get_llm_provider raise litellm.exceptions.BadRequestError( # type: ignore litellm.exceptions.BadRequestError: litellm.BadRequestError: GetLLMProvider Exception - 'ChatGoogleGenerativeAI' object has no attribute 'split'
> 
> original model: verbose=True model='models/gemini-1.5-flash' google_api_key=SecretStr('**********') temperature=0.5 client=<google.ai.generativelanguage_v1beta.services.generative_service.client.GenerativeServiceClient object at 0x000002289050DAB0> default_metadata=()

did u get the solution to that error? im facing the same error...

**Comment by sobit-nep** [comment]
yeah..i solved it. You can check my projects on my profile: https://github.com/sobit-nep/cv-to-portfolio-crewai-agent

**Comment by dyno-star** [comment]
Make sure you used the proper configuration
